lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -275,6 +275,17 @@ export abstract class CacheManager implements ICacheManager {
         accountFilter: AccountFilter,
         correlationId: string
     ): AccountInfo | null {
+        if (
+            Object.keys(accountFilter).length === 0 ||
+            Object.values(accountFilter).every(
+                (value) => value === null || value === undefined
```

Comment:
More concise and captures empty string as well
```suggestion
                (value) => !value
```